calico-policies/private-network-isolation/calico-v2/allow-vrrp-private.yaml

```diff
@@ -0,0 +1,16 @@
+# This policy opens the VRRP protocol, which manages load balancing.
```

Comment:
I would say "This policy opens the VRRP protocol, which is required when running kubernetes LoadBalancer services"